### PR TITLE
ISSUE-80 Stanislav Gorbunov

### PR DESCRIPTION
**Pull request by lamertoprogrammer**

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -77,8 +77,8 @@
 30 + 86
 72 + 78
 37 - 75
-48 - 40
-74 + 87
+48 - 40 = 23232
```
Comment (ekirillm) [inline tasks]: Solution is not correct

Comment (lamertoprogrammer) [reply tasks]: Thank

Comment (lamertoprogrammer) [reply tasks]: Fix it

Comment (lamertoprogrammer) [reply tasks]: Fix it

**Pull request merged**